src/main/java/core/basesyntax/Triangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+class RightTriangle extends Figure {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line at the beginning of the class definition.